packages/tailwind/package.json

```diff
@@ -19,12 +20,21 @@
     "src",
     "dist"
   ],
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
   "author": "",
   "private": false,
   "license": "MIT",
   "dependencies": {
-    "@kaizen/design-tokens": "workspace:*"
+    "@kaizen/design-tokens": "workspace:*",
+    "@kaizen/components": "workspace:*"
```

Comment:
I'm a bit wary of this as a dep. KAIO has this package as a dep too doesn't it? So it would become circular.